kubeflow/common/kubeflow-issuer/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+# Define the self-signed issuer for Kubeflow
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: cert-manager
+resources:
+- ../cert-manager/cert-manager-1-3/cert-manager/kubeflow-issuer
```

Comment:
nit: simply move that folder here?

Reply:
I am a bit hesitant to move an upstream folder to another folder. It will make future version upgrade a bit complicated. Another solution is to build a Makefile under common/cert-manager which will build both cert-manager and kubeflow-issuer. How do you think?

Reply:
Adding this to cert-manager make sense, as that's how upstream organizes the folder.

Reply:
@zijianjoy I am strongly against any references between packages, because goal of this refactor is to make them independent.

Reply:
Sounds good, it makes sense to me to establish the independency between packages. I have merged the kubeflow-issuer to cert-manager as in https://github.com/kubeflow/gcp-blueprints/pull/238/files.